keyboards/planck/keymaps/rootiest/keymap.c

```diff
@@ -0,0 +1,1920 @@
+/**
+ * Copyright (c) 2021 Chris Laprade (chris@rootiest.com)
+ *
+ * This software is released under the MIT License.
+ * https://opensource.org/licenses/MIT
+ */
+
+#include QMK_KEYBOARD_H
+#include "raw_hid.h"
+#include "print.h"
+#include "muse.h"
+
+/*
+ * Define keyboard name to be used by UI.
+ * This allows for using a different name
+ * than the one defined by QMK for your board.
+ */
+#define KEEB_MODEL_NAME "The Rootiest BoogieBoard"
+
+// Give names to the layers so they are easier to reference
+#define _BASE 0
+#define _QWERTY 0
+#define _COLEMAK 1
+#define _DVORAK 2
+#define _SYMBLS 3
+#define _RAISE 3
+#define _NUMBRS 4
+#define _LOWER 4
+#define _PLOVER 5
+#define _FEATURS 6
+#define _ADJUST 6
+#define _NUMPD 7
+#define _TABULA 8
+#define _MOUSY 9
```

Comment:
We highly recommend using enums for layer naming, as it's simpler to maintain, long term. 
```suggestion
enum custom_layers {
    _BASE,
    _QWERTY = _BASE,
    _COLEMAK,
    _DVORAK,
    _SYMBLS,
    _RAISE = _SYMBLS,
    _NUMBRS,
    _LOWER = _NUMBRS,
    _PLOVER,
    _FEATURS,
    _ADJUST = _FEATURS,
    _NUMPD,
    _TABULA,
    _MOUSY,
};
```